yarn.lock

```diff
@@ -1259,6 +1259,11 @@ acorn@^6.4.1:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.4.1.tgz#531e58ba3f51b9dacb9a6646ca4debf5b14ca474"
   integrity sha512-ZVA9k326Nwrj3Cj9jlh3wGFutC2ZornPNARZwsNYqQYgN0EsV2d53w5RN/co65Ohn4sUAUtb1rSUAOD6XN9idA==
 
+after@0.8.2:
```

Comment:
Where did these new deps come from? Are they new transitive dependencies?

Reply:
From running yarn on my client. I don't know why, I didn't change package.json.
Perhaps yarn.lock in master was not updated last time package.json changed.